### ISSUE TITLE
Infrastructure for keeping a11y rules folder content in sync with README markdown content

### ISSUE THREAD

**Issue by uncommon-type** [jsx-eslint/eslint-plugin-jsx-a11y#836]
AC:
* `README` `Supported Rules` list and `Rule strictness in different modes` table should be generated programmatically
* The list and table should be kept in sync with a11y rules in the `/rules` folder. Every time a new a11y rule is added to `/rules` folder , `README` list and table content should be refreshed programmatically
